### PR TITLE
Problem: (CRO-562) tiny-keccak 2.0 fails to compile

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Made necessary changes for `tiny-keccak = 2.0`

### CODE REVIEW

File: chain-core/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ mesalock_sgx = ["sgx_tstd", "secp256k1zkp/sgx"]
 
 [dependencies]
 digest = { version = "0.8", default-features = false}
-tiny-keccak = { version = "1.5.0", default-features = false, features = ["keccak"] }
+tiny-keccak = { version = "2.0", features = ["keccak"] }
```
Comment (tomtau) [inline chain-core/Cargo.toml]: is it `no_std` now? `default-features = false` was there before due to std, if i remember correctly

**Comment by tomtau** [comment]
bors try

File: chain-core/Cargo.toml (hunk 1 above)
Comment (devashishdxt) [reply chain-core/Cargo.toml]: Yes. It is now `no_std`.

**Comment by tomtau** [comment]
bors cancel

**Pull request merged**